src/MongoDB.EntityFrameworkCore/Serializers/SerializationHelper.cs

```diff
@@ -148,6 +148,22 @@ private static IBsonSerializer CreateTypeSerializer(Type type)
         return serializer;
     }
 
+    private static IBsonSerializer CreateDateTimeSerializer(IReadOnlyProperty? property)
+    {
+        var dateTimeKind = DateTimeKind.Unspecified;
+        if (property != null)
+        {
+            dateTimeKind = property.GetDateTimeKind();
+        }
+
+        if (dateTimeKind == DateTimeKind.Unspecified)
+        {
+            return new DateTimeSerializer();
```

Comment:
Something about the default for DateTimeSerializer being DateTimeKind.Unspecified while the convention defaults to DateTimeKind.Utc is feeling off. Perhaps we should make the serializer constructor require a DateTimeKind and remove this block?

Reply:
We still need this block, because passing `DateTimeKind.Unspecified` will works same way as Local, but parameterless constructor of `DateTimeSerializer` uses Utc as default value. I would prefer to follow the MongoDB driver's defaults.